### ISSUE TITLE
Advice on improving accuracy of the TFRS examples

### ISSUE THREAD

**Issue by cfregly** [tensorflow/recommenders#97]
After running all of the examples, i'm only seeing 20-25% accuracy.

Any tips on improving the accuracy?  Is there a different dataset that we should use to highlight the awesomeness of this framework?

**Comment by maciejkula** [comment]
On the most basic level, accuracy can be improved via better model architectures and hyperparameter tuning: all the standard tools from the experimenter's toolbox. 

Our next release will, however, contain a few additional modelling techniques that can lead to substantial gains in accuracy. Stay tuned :)

On the dataset question, the MovieLens datasets are the unit tests of recommender models and are great for this purpose. However, they lack the complexity or rich features of real-world applications. Perhaps something more complex like the [Criteo click dataset](http://labs.criteo.com/2013/12/download-terabyte-click-logs/) might help?
